src/tools/tool-options.h

```diff
@@ -134,7 +134,7 @@ struct ToolOptions : public Options {
   void applyFeatures(Module& module) {
     if (hasFeatureOptions) {
       if (!detectFeatures && module.hasFeaturesSection) {
-        FeatureSet optionsFeatures = FeatureSet::MVP;
+        FeatureSet optionsFeatures;
```

Comment:
Hmm, this changes not just the default for the tool but also the interpretation of the target features section. Can we hold off on that part of the change for now until we figure out a policy for changing its meaning at the tool-conventions level?